ro_help/hub/templatetags/social_share.py

```diff
@@ -0,0 +1,22 @@
+from django import template
+
+TWITTER_ENDPOINT = "https://twitter.com/intent/tweet?text=%s"
+FACEBOOK_ENDPOINT = "https://www.facebook.com/sharer/sharer.php?u=%s"
+LINKEDIN_ENDPOINT = "https://www.linkedin.com/sharing/share-offsite/?url=%s"
+
+register = template.Library()
+
+
+def _build_full_url(request, obj):
```

Comment:
Nice! This one can be a separate util file (`hub/utils.py`). It can be used in another context as well.